.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
same comment as before, can we put these in the .github repository and have the same templates across all projects?